WcaOnRails/db/migrate/20160504230105_add_results_posted_to_competitions.rb

```diff
@@ -0,0 +1,14 @@
+class AddResultsPostedToCompetitions < ActiveRecord::Migration
+  def up
+    add_column :Competitions, :results_posted, :datetime
+    execute <<-SQL
+      UPDATE Competitions
+      SET results_posted = (SELECT max(updated_at) FROM Results)
+      WHERE results_posted is NULL AND id IN (SELECT competitionId FROM Results)
```

Comment:
Ah, I misread it. I'll update to maybe
      UPDATE Competitions
      SET results_posted = (SELECT max(updated_at) FROM Results where competitionId = Competitions.id GROUP BY competitionId)
      WHERE results_posted is NULL AND id IN (SELECT competitionId FROM Results)